tests/languages/rescript/comment_feature.test

```diff
@@ -0,0 +1,22 @@
+// foobar
```

Comment:
The comment pattern also includes stuff like

```
// foo\
bar
```

Is that correct? If so, please add a test case.

Reply:
I'm not sure, but probably doesn't include this kind of stuff.

Reply:
The docs say that single line comments are the same as JS, so just `//.*` for single line comments should be fine. We don't need the whole backslash thing.

Reply:
You're right!  I'll fix

Reply:
Done